packages/fluent-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -71,9 +67,6 @@ export default function CheckboxWidget<
     [onChange]
   );
 
-  const _onBlur = ({ target: { value } }: FocusEvent<HTMLButtonElement>) => onBlur(id, value);
-  const _onFocus = ({ target: { value } }: FocusEvent<HTMLButtonElement>) => onFocus(id, value);
-
```

Comment:
Does fluent not support onBlur and onFocus? This could cause some issues with some users

Reply:
Sorry about this one! I was working on this in a bit of a haste then and since force installing the latest package in my project worked, I didn't really pay attention to what changed.

These properties (all native input props) have been moved into an `inputProps` prop starting from v8. Updated this is my latest commit.